data/individual_score_test.json

```diff
@@ -0,0 +1,10337 @@
+{
```

Comment:
I'm not sure what the intention behind creating this file is. We already have a json file populated with some sample data that is smaller and easier to look at compared to this very large one. I suggest either using that file or trying temporary directories as an alternative.

Reply:
We created this file to test our program, however, it should not be merged with master branch so should be removed